libs/designsystem/src/lib/components/highstock-chart/highstock-chart.component.ts

```diff
@@ -0,0 +1,88 @@
+import { Component, ElementRef, Inject, Input, LOCALE_ID, ViewChild } from '@angular/core';
+
+import { Options } from 'highcharts';
+import * as Highcharts from 'highcharts/highstock';
```

Comment:
Consider only importing what's used:
```suggestion
import { stockChart } from 'highcharts/highstock';
```

Reply:
Done.